### PR TITLE
Login and Signup Screens

### PR DESCRIPTION
**Pull request by talhamujahid52**

### CODE REVIEW

File: app/(auth)/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        <Text style={styles.Col1}>Don’t have an account?</Text>
+        <TouchableOpacity
+          onPress={() => {
+            router.push({
+              pathname: "/(auth)/Register",
+            });
+          }}
+        >
+          <Text style={styles.Col2}> Register here.</Text>
+        </TouchableOpacity>
```
Comment (shariqwaseem) [inline app/(auth)/Login.tsx]: This should be a variant of button component

File: app/(auth)/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        title="Login"
+        onPress={() => {
+          router.push({
+            pathname: "/(auth)/Register",
+          });
+        }}
+      />
+      <View style={styles.SpacerContainer}>
+        <View style={styles.Spacer}></View>
+        <Text style={styles.SpacerText}>Or continue with</Text>
```
Comment (shariqwaseem) [inline app/(auth)/Login.tsx]: Text should be a separate component not imported from React native

Comment (talhamujahid52) [reply app/(auth)/Login.tsx]: Done.

File: app/(auth)/Login.tsx (hunk 1 above)
Comment (talhamujahid52) [reply app/(auth)/Login.tsx]: Will change in a latter commit.

**Review by shariqwaseem — APPROVED** [review]

**Pull request merged**